digdag-standards/src/main/java/io/digdag/standards/operator/MailOperatorFactory.java

```diff
@@ -91,23 +119,11 @@ public TaskResult runTask(TaskExecutionContext ctx)
         {
             SecretProvider secrets = ctx.secrets().getSecrets("mail");
 
-            Config bodyParams = request.getConfig().mergeDefault(
+            Config params = request.getConfig().mergeDefault(
                     request.getConfig().getNestedOrGetEmpty("mail"));
 
-            // Note: Do not include system mail config params in the body template params to
-            //       ensure that they are not accessible to the user.
-            String body = workspace.templateCommand(templateEngine, bodyParams, "body", UTF_8);
-
-            // Load system mail config params
-            Config params = bodyParams.deepCopy();
-            String configPrefix = "config.mail.";
-            systemConfig.getKeys().stream()
-                    .filter(key -> key.startsWith(configPrefix))
-                    .forEach(key -> params.setIfNotSet(
-                            key.substring(configPrefix.length()),
-                            systemConfig.get(key, String.class)));
-
-            String subject = params.get("subject", String.class);
+            String body = workspace.templateCommand(templateEngine, params, "body", UTF_8);
+            String subject = params.getOptional("subject", String.class).or(mailDefaults.subject()).or(() -> params.get("subject", String.class));
```

Comment:
To throw this exception: https://github.com/treasure-data/digdag/blob/master/digdag-client/src/main/java/io/digdag/client/config/Config.java#L232